db/migrate/20220915125708_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :authorId
+      t.string :title
+      t.text :text
+      t.integer :commentsCounter
+      t.integer :likeCounter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- As mentioned above, kindly make sure Foreign key is included with the corresponding index 👍🏽